pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -6,1413 +6,1677 @@
+  }
+
+  .zero-top-margin {
     margin-top: 0;
-}
-.zero-bottom-padding{
+  }
+
+  .zero-bottom-padding {
     padding-bottom: 0;
-}
-.side-padding{
+  }
+
+  .side-padding {
     padding-left: 30px;
     padding-right: 30px;
-}
-.bottom-margin{
+  }
+
+  .bottom-margin {
     margin-bottom: 30px;
-}
-.bottom-padding{
+  }
+
+  .bottom-padding {
     padding-bottom: 30px;
-}
-.top-padding{
+  }
+
+  .top-padding {
     padding-top: 30px;
-}
-.bottom-margin{
+  }
+
+  .bottom-margin {
     margin-bottom: 30px;
-}
-.top-margin{
+  }
+
+  .top-margin {
     margin-top: 30px;
-}
-/*.bottom-border{
+  }
+
+  /*.bottom-border{
     border-bottom-width: 1px;
     border-bottom-color: #ddd;
     border-bottom-style: solid;
 }*/
-.transparent-background{
+  .transparent-background {
     background-color: transparent;
-}
-.code-row{
+  }
+
+  .code-row {
     border-color: #ddd;
     border-width: 1px;
     border-radius: 4px 4px 0 0;
     border-style: solid;
     padding: 15px 15px 15px;
     background-color: white;
-}
-.modal-body{
+  }
+
+  .modal-body {
     max-height: 500px;
     overflow: auto;
-}
-.github-logo{
+  }
+
+  .github-logo {
     float: left;
     margin-top: 10px;
     max-width: 100px;
     max-height: 100px;
-}
-#top-link-block.affix-top{
+  }
+
+  #top-link-block.affix-top {
     position: absolute;
-}
+  }
 </style>
 
 
-<div class="modal fade" id="step-collection-modal" tabindex="-1" role="dialog" aria-labelledby="step-collection-modal-label" aria-hidden="true">
-<div class="modal-dialog modal-lg">
-<div class="modal-content">
-<div class="modal-header">
-<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button></p>
-<h4 class="modal-title" id="step-collection-modal-label">Response</h4>
-</p></div>
-<div class="modal-body">
-{% highlight js %}
-HTTP/1.1 200 OK
-Content-Length: 1007
-Content-Type: application/json; odata.metadata=minimal
-OData-Version: 4.0
-{
-'@odata.context': 'https://services.odata.org/V4/(S(a2k31bgwiyejn2j2iiybvq4p))/TripPinServiceRW/$metadata#People',
-'@odata.nextLink': 'https://services.odata.org/V4/(S(a2k31bgwiyejn2j2iiybvq4p))/TripPinServiceRW/People?%24skiptoken=8',
-'value': [
-    {
-        '@odata.id': 'https://services.odata.org/V4/(S(a2k31bgwiyejn2j2iiybvq4p))/TripPinServiceRW/People('russellwhyte')',
+<div class="modal fade" id="step-collection-modal" tabindex="-1" role="dialog"
+  aria-labelledby="step-collection-modal-label" aria-hidden="true">
+  <div class="modal-dialog modal-lg">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span
+            class="sr-only">Close</span></button></p>
+        <h4 class="modal-title" id="step-collection-modal-label">Response</h4>
+        </p>
+      </div>
+      <div class="modal-body">
+        {% highlight js %}
+        HTTP/1.1 200 OK
```

Comment:
Will this leading indentation also appear on the rendered page? Or will `highlight` take care of that?

Reply:
Nice catch